week-3-pratise/exercise3.js

```diff
@@ -0,0 +1,14 @@
+/* Given is an array return an array containing only even numbers */
+
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+var filterNumber = numbers.filter(function(number) {
+  if (number % 2 === 0) {
+    return number;
```

Comment:
This has the same problem as the other `.filter()` exercise - you need to return a boolean, not the element itself.